File: 01.JS-Basic/По сложни проверки/TradeCommissions.js
function tradeCommissions(input) {
    let town = input[0]
    let s = Number(input[1])

    if (s >= 0 && s <= 500) {

        switch (town) {
            case "Sofia": console.log((s * 0.05).toFixed(2)); break;
            case "Varna": console.log((s * 0.045).toFixed(2)); break;
            case "Plovdiv": console.log((s * 0.055).toFixed(2)); break;
            default: console.log("error");break;
        }

    }
    else if (s > 500 && s <= 1000) {

        switch (town) {
            case "Sofia": console.log((s * 0.07).toFixed(2)); break;
            case "Varna": console.log((s * 0.075).toFixed(2)); break;
            case "Plovdiv": console.log((s * 0.08).toFixed(2)); break;
            default: console.log("error");break;
        }
    }
    else if (s > 1000 && s <= 10000) {

        switch (town) {
            case "Sofia": console.log((s * 0.08).toFixed(2)); break;
            case "Varna": console.log((s * 0.1).toFixed(2)); break;
            case "Plovdiv": console.log((s * 0.12).toFixed(2)); break;
            default: console.log("error");break;
        }
    }
    else if (s > 10000) {

        switch (town) {
            case "Sofia": console.log((s * 0.12).toFixed(2)); break;
            case "Varna": console.log((s * 0.13).toFixed(2)); break;
            case "Plovdiv": console.log((s * 0.145).toFixed(2)); break;
            default: console.log("error");break;
        }
    }
    else { console.log("error") }
}
tradeCommissions(["Kaspichan",
"-50"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatementsAdvanced/TradeCommissions.java
package ConditionalStatementsAdvanced;

import java.util.Scanner;

public class TradeCommissions {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String town = scan.nextLine();
        double s = Double.parseDouble(scan.nextLine());

        switch (town) {
            case "Sofia":
                if (s < 0) {
                    System.out.println("error");
                } else if (s <= 500) {
                    System.out.printf("%.2f", s * 0.05);
                } else if (s <= 1000) {
                    System.out.printf("%.2f", s * 0.07);
                } else if (s <= 10000) {
                    System.out.printf("%.2f", s * 0.08);
                } else if (s > 10000) {
                    System.out.printf("%.2f", s * 0.12);
                } else {
                    System.out.println("error");
                }
                break;
            case "Varna":
                if (s < 0) {
                    System.out.println("error");
                } else if (s <= 500) {
                    System.out.printf("%.2f", s * 0.045);
                } else if (s <= 1000) {
                    System.out.printf("%.2f", s * 0.075);
                } else if (s <= 10000) {
                    System.out.printf("%.2f", s * 0.10);
                } else if (s > 10000) {
                    System.out.printf("%.2f", s * 0.13);
                } else {
                    System.out.println("error");
                }
                break;
            case "Plovdiv":
                if (s < 0) {
                    System.out.println("error");
                } else if (s <= 500) {
                    System.out.printf("%.2f", s * 0.055);
                } else if (s <= 1000) {
                    System.out.printf("%.2f", s * 0.08);
                } else if (s <= 10000) {
                    System.out.printf("%.2f", s * 0.12);
                } else if (s > 10000) {
                    System.out.printf("%.2f", s * 0.145);
                } else {
                    System.out.println("error");
                }
                break;
            default:
                System.out.println("error");
        }
    }
}
